guides/release/routing/specifying-a-routes-model.md

```diff
@@ -17,11 +17,8 @@ Here's an example of a model hook in use within a route:
 
 ```javascript {data-filename=app/routes/favorite-posts.js}
 import Route from '@ember/routing/route';
-import { inject as service } from '@ember/service';
 
 export default class FavoritePostsRoute extends Route {
-  @service store;
```

Comment:
It looks like a store injection may have been removed from a cut-paste. Will you revert the change in the respective folders?

Reply:
@ijlee2 Yea, I removed it because its redundant here, we do not use store in that case

Reply:
@ijlee2 please re-verify

Reply:
Ah, I see it now. Thanks for clarifying!